triggers/daml/Daml/Trigger.daml

```diff
@@ -199,6 +200,33 @@ emitCommands cmds pending = do
     }
   pure id
 
+-- Version of emitCommands that will fail when there are too many commands in-flight.
+-- When this function fails to return a command ID, then client code should manage this
+-- scenario. Failing to manage this warning scenario will eventually cause the trigger
+-- runner to stop with an InFlightCommandOverflowException exception.
```

Comment:
Maybe we could use a word less strong that "fail". Maybe swallow.
not 100% sure, more a suggestion.